examples/flakes/flake.nix

```diff
@@ -0,0 +1,34 @@
+{
+  description = "A basic nix-bitcoin node";
+
+  inputs.nix-bitcoin.url = "github:fort-nix/nix-bitcoin";
+
+  outputs = { self, nix-bitcoin }: {
+
+    nixosConfigurations.mynode = nix-bitcoin.inputs.nixpkgs.lib.nixosSystem {
```

Comment:
Our README states the goal that nix-bitcoin enables privacy by default. That's not the case in this example because we don't enable the secure-node preset and hence the traffic isn't torified. I think it's worth mentioning that fact (same applies to the minimal config actuall) and show how to enable the secure node preset.